webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/GeneralScrambleSheet.kt

```diff
@@ -33,32 +32,27 @@ class GeneralScrambleSheet(scrambleSet: ScrambleSet, activityCode: ActivityCode)
         val maxScrambleImageHeight = (allScramblesHeight / scramblesPerPage - 2 * SCRAMBLE_IMAGE_PADDING).toInt()
 
         // We don't let scramble images take up too much of a the page
-        val maxScrambleImageWidth = (availableWidth / 2).toInt().takeUnless {
-            // TODO - If we allow the megaminx image to be too wide, the
-            //   Megaminx scrambles get really tiny. This tweak allocates
-            //   a more optimal amount of space to the scrambles. This is possible
-            //   because the scrambles are so uniformly sized.
-            scramblingPuzzle.shortName == "minx"
-        } ?: 190
+        val maxScrambleImageWidth = (availableWidth / 3).toInt()
```

Comment:
Where did you take that "a third of available width" from (as opposed to the previous "half of"), and how does it relate to the 190 constant for Megaminx?

Reply:
190 was a hack introduced by someone. Half of it just sound a lot. A third sounds the correct choice. I could remove the 190 hack by splitting in new lines first.

Reply:
I added a const on the companion, but 3 is just an arbitrary choice, like 2 was (and I think an imagem filling half of the width would be disturbing to the eye).